tests/test_downloader_handlers.py

```diff
@@ -437,6 +437,8 @@ class S3AnonTestCase(unittest.TestCase):
         import boto
     except ImportError:
         skip = 'missing boto library'
+    if six.PY3:
+        skip = 'S3 not supported on Py3'
```

Comment:
Why isn't it supported? Is it because we haven't ported relevant code in Scrapy yet, or are there other reasons (e.g. some dependency)?